Wise/pom.xml

```diff
@@ -16,8 +16,8 @@
     <hadoop.version>[2.0.2-alpha, 2.3.0]</hadoop.version>
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <co.cask.cdap.version>3.1.0</co.cask.cdap.version>
-  </properties>
+    <co.cask.cdap.version>3.2.0-SNAPSHOT</co.cask.cdap.version>
```

Comment:
for consistency, please change this property name to `cdap.version`.
Otherwise, LGTM.